package.json

```diff
@@ -51,12 +51,12 @@
     "golden-layout": "1.5.9",
     "iodide-react-inspector": "2.3.2",
     "jss": "9.8.7",
+    "katex": "^0.10.1",
```

Comment:
I didn't commit the `package-lock.json`, because npm fucks everything up in the lock-file as soon as I remove or install a package. (I guess the npm-version has to match...)